pkg/gui/gui.go

```diff
@@ -625,7 +628,8 @@ func (gui *Gui) runSubprocess(subprocess *exec.Cmd) error {
 
 	fmt.Fprintf(os.Stdout, "\n%s\n\n", style.FgBlue.Sprint("+ "+strings.Join(subprocess.Args, " ")))
 
-	if err := subprocess.Run(); err != nil {
+	err := subprocess.Run()
+	if err != nil {
 		// not handling the error explicitly because usually we're going to see it
```

Comment:
we can remove this part because we are now indeed explicitly handling the error

Reply:
Done